pkg/limits/frontend/frontend_test.go

```diff
@@ -171,28 +177,31 @@ func TestFrontend_ExceedsLimits(t *testing.T) {
 		// and instance 1 for the data for partition 1.
 		expectedStreamUsageRequest: []*logproto.GetStreamUsageRequest{{
 			Tenant:       "test",
-			StreamHashes: []uint64{0x1, 0x2},
+			StreamHashes: []uint64{0x2},
 		}, {
 			Tenant:       "test",
-			StreamHashes: []uint64{0x1, 0x2},
+			StreamHashes: []uint64{0x1},
 		}},
-		// Each instance will respond stating that it doesn't know about the
```

Comment:
This is no longer true, since we stopped sending all stream hashes to all instances.